modules/ipi-install-establishing-communication-between-subnets.adoc

```diff
@@ -6,16 +6,29 @@
 [id="ipi-install-establishing-communication-between-subnets_{context}"]
 = Establishing communication between subnets
 
-In a typical {product-title} cluster setup, all nodes, including the control plane and worker nodes, reside in the same network. However, for edge computing scenarios, it can be beneficial to locate worker nodes closer to the edge. This often involves using different network segments or subnets for the remote worker nodes than the subnet used by the control plane and local worker nodes. Such a setup can reduce latency for the edge and allow for enhanced scalability. However, the network must be configured properly before installing {product-title} to ensure that the edge subnets containing the remote worker nodes can reach the subnet containing the control plane nodes and receive traffic from the control plane too.
+In a typical {product-title} cluster setup, all nodes, including the control plane and worker nodes, reside in the same network. However, for edge computing scenarios, it can be beneficial to locate worker nodes closer to the edge. This often involves using different network segments or subnets for the remote worker nodes than the subnet used by the control plane and local worker nodes. Such a setup can reduce latency for the edge and allow for enhanced scalability.
+
+Before installing {product-title}, you must configure the network properly to ensure that the edge subnets containing the remote worker nodes can reach the subnet containing the control plane nodes and receive traffic from the control plane too.
+
+You can run control plane nodes in the same subnet or multiple subnets by configuring a user-managed load balancer in place of the default load balancer. If you run a multiple subnet environment, you can reduce the risk of your {product-title} cluster from failing due to a hardware failure or a network outage. For more information, see "Services for a user-managed load balancer" and "Configuring a user-managed load balancer".
 
 [IMPORTANT]
 ====
-All control plane nodes must run in the same subnet. When using more than one subnet, you can also configure the Ingress VIP to run on the control plane nodes by using a manifest. See "Configuring network components to run on the control plane" for details.
+If you run your control plane nodes on a multiple subnet environment, you must specify a different Ingress VIP address or addresses for remote worker nodes.
```

Comment:
This is no longer specific to the ingress VIP. For cross-subnet control plane both the API and Ingress VIPs need to point at the user-managed LB, so I think this note should be modified to say that.

Alternatively, we could remove this completely since we now discuss the requirements for multiple subnet envs below.

Reply:
Thanks for stating that as I was about to ask. Yes, it's covered below but worded differently. I'll remove the note.